docker/pi0neerpat-packages/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '3'
+services:
+  database:
+    image: 'postgres:latest'
+    # NOTE: avoid collision with local postgres port
+    ports:
+      - 6543:5432
+    environment:
+      POSTGRES_USER: username
+      POSTGRES_PASSWORD: password
+      POSTGRES_DB: default_database
+  api:
+    image: pi0neerpat/redwood-devops-example-api:latest
```

Comment:
This might want to be changed later when the image is built.

Reply:
Agreed. Any thoughts on that? Using local packages is the only issue here, which would prevent the other images from building properly.